base/codeintel-db/codeintel-db.Deployment.yaml

```diff
@@ -20,6 +20,8 @@ spec:
     type: Recreate
   template:
     metadata:
+      annotations:
+        kubectl.kubernetes.io/default-container: pgsql
```

Comment:
Ah, looks like we had added these before but in the wrong place. Should the annotation be removed from [L6](https://github.com/sourcegraph/deploy-sourcegraph/pull/4047/files#diff-b822422461c5c4358eb788df612e854aa5d5701144feebb0b863c0655fe6a2f5L6)? (and same for all the other deployments)